Assets/Scripts/PlayerManager.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerManager : MonoBehaviour
+{
+    #region Singleton
+    private static PlayerManager _instance;
+    public static PlayerManager Instance { get { return _instance; } }
```

Comment:
If we start with couple of Singleton-Managers we should probably make a Singleton class. Also:
`public static PlayerManager Instance { get; private set; } `
would do the same thing, as the field and property.